app/i18next-parser.config.js

```diff
@@ -0,0 +1,9 @@
+const configs = require('../i18next-parser.config');
+
+/*eslint no-undef: "error"*/
+/*eslint-env node*/
```

Comment:
are this intentional?

Reply:
@p-netm No they are not...I think I must have copied them from the prev file